config/install/user.role.localgov_news_editor.yml

```diff
@@ -15,26 +15,12 @@ permissions:
   - 'create image media'
   - 'create localgov_news_article content'
   - 'delete any localgov_news_article content'
+  - 'delete own image media'
   - 'delete own localgov_news_article content'
-  - 'delete own media'
   - 'edit any image media'
   - 'edit any localgov_news_article content'
   - 'edit own image media'
   - 'edit own localgov_news_article content'
   - 'revert localgov_news_article revisions'
-  - 'update any image media'
-  - 'use localgov_editorial transition approve'
-  - 'use localgov_editorial transition archive'
-  - 'use localgov_editorial transition archived_draft'
-  - 'use localgov_editorial transition archived_published'
-  - 'use localgov_editorial transition create_new_draft'
-  - 'use localgov_editorial transition publish'
-  - 'use localgov_editorial transition reject'
-  - 'use localgov_editorial transition submit_for_review'
-  - 'view all localgov_news_article revisions'
-  - 'view any unpublished localgov_news_article content'
```

Comment:
Removing the permissions from the other modules I get, but the view unpublished localgov_news ? It does seem to be a permission drupal understands?

Reply:
There aren't permissions `view any unpublished localgov_news_article content` or `view own unpublished localgov_news_article content` included with LocalGov Drupal, so I've removed these permissions. I guess they came from another module, but this isn't being installed so they're not needed.

Is this the only thing that needs resolving @ekes?